api/interservice/infra_proxy/response/cookbooks.proto

```diff
@@ -4,16 +4,25 @@ package chef.automate.domain.infra_proxy.response;
 option go_package = "github.com/chef/automate/api/interservice/infra_proxy/response";
 
 message Cookbooks {
+    // List of cookbooks with name and version.
     repeated CookbookVersion cookbooks = 1;
 }
 
-message CookbooksAvailableVersions {
-    repeated CookbookAllVersion cookbooks = 1;
+message CookbookVersions {
+    // List of all versions avaiable for cookbook.
+    repeated string versions = 1;
```

Comment:
Added `versions` attributes to return only the list of the version available for a particular cookbook